Source/WebCore/rendering/updating/RenderTreeUpdaterViewTransition.cpp

```diff
@@ -121,17 +121,25 @@ static RenderPtr<RenderBox> createRendererIfNeeded(RenderElement& documentElemen
         return nullptr;
 
     Ref document = documentElementRenderer.document();
-    auto newStyle = RenderStyle::clone(*style);
     RenderPtr<RenderBox> renderer;
     if (pseudoId == PseudoId::ViewTransitionOld || pseudoId == PseudoId::ViewTransitionNew) {
+        const auto* capturedElement = document->activeViewTransition()->namedElements().find(name);
```

Comment:
It seems we can crash if `capturedElement` is `nullptr` here.

Either check for this, or ASSERT it's not if it's an impossible state.